platform/android/java/editor/src/main/AndroidManifest.xml

```diff
@@ -29,11 +29,12 @@
             android:name=".GodotProjectManager"
             android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|density|keyboard|navigation|screenLayout|uiMode"
             android:launchMode="singleTask"
-            android:resizeableActivity="false"
+            android:resizeableActivity="true"
```

Comment:
Here and below, you can remove the `android:resizeableActivity` attribute. It defaults to `true` so there's no need to repeat it here.